aws_neuron/tests/common.py

```diff
@@ -0,0 +1,63 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+
+from datadog_checks.dev import get_docker_hostname, get_here
+
+HERE = get_here()
+HOST = get_docker_hostname()
+PORT = 8000
+
+
+def get_fixture_path(filename):
+    return os.path.join(HERE, 'fixtures', filename)
+
+
+MOCKED_INSTANCE = {
+    "openmetrics_endpoint": f"http://{HOST}:{PORT}/metrics",
+    'tags': ['test:tag'],
+}
+
+COMPOSE_FILE = os.path.join(HERE, 'docker', 'docker-compose.yaml')
+
+TEST_METRICS = [
+    'aws_neuron.execution.errors_created',
```

Comment:
If you wanna be more ~anal~ thorough you could also add metric types here and assert those.